tinyagentos/routes/project_files.py

```
@@ -0,0 +1,213 @@
+    return StreamingResponse(
+        event_stream(),
+        media_type="text/event-stream",
+        headers={
+            "Cache-Control": "no-cache",
+            "X-Accel-Buffering": "no",
+        },
+    )
+
+
+@router.post("/api/projects/{slug}/files/upload")
+async def api_project_upload_file(request: Request, slug: str, path: str = "", file: UploadFile = File(...)):
```

Ruff (0.15.11)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)